.github/workflows/helm-test.yaml

```diff
@@ -94,3 +93,11 @@ jobs:
       - name: Run chart-testing (install)
         run: ct install --charts charts/data-exchange --config charts/chart-testing-config.yaml --helm-extra-set-args "--set=image.tag=testing --set=image.repository=kind-registry:5000/data-exchange-test-service"
         if: ${{ env.CHART_CHANGED == 'true' }}
+              # Upgrade the released chart version with the locally available chart
```

Comment:
The indentation seems to be too deep here

Reply:
Corrected.